PowerShell/ScubaGear/Rego/AADConfig.rego

```diff
@@ -26,9 +26,9 @@ import data.utils.aad.INT_MAX
 # Constants #
 #############
 
-RESTRICTEDACCESS := "2af84b1e-32c8-42b7-82bc-daa82404023b"
+RESTRICTEDACCESS := "2af84b1e-32c8-42b7-82bc-daa82404023b" #gitleaks:allow
```

Comment:
Interesting that it is rather simplistic -- I suspect ACCESS triggered as secret.

Reply:
That seems possible, because it did not trigger on MEMBERUSER whose value was an identical format as RESTRICTEDACCESS and LIMITEDACCESS.  Perhaps it is simply finding a common user mistake.